### PR TITLE
PEP 584: Apply feedback from the Python-Dev discussion.

### PR DESCRIPTION
**Pull request by brandtbucher**
These are just a collection of clarifications that were requested on- and off-list.

### CODE REVIEW

File: pep-0584.rst [hunk 1]
```diff
@@ -166,21 +191,24 @@ An *approximate* pure-Python implementation is::
     def __or__(self, other):
         if not isinstance(other, dict):
             return NotImplemented
-        new = self.copy()
+        new = dict.copy(self)
```
Comment (gvanrossum) [inline pep-0584.rst]: I typically spell this `dict(self)`.

File: pep-0584.rst [hunk 2]
```diff
@@ -82,6 +82,29 @@ ChainMap will modify the original dict::
...
+      ...
+    TypeError: keywords must be strings
+
+Even then, it should not be promoted as the "one obvious way".  Guido
+`said it best
+<https://mail.python.org/archives/list/python-dev@python.org/message/FM66WWN5YFCMRGMRWGHACOXLQYTWNXE2/>`_:
+
+    Apparently dict(x, \*\*y) is going around as "cool hack" for "call
+    x.update(y) and return x". Personally I find it more despicable
+    than cool.
```
Comment (gvanrossum) [inline pep-0584.rst]: I don't think my emotional response adds much insight to the PEP (I also misstated the trick). I think the fact that it only works for string keys is enough to sink it.

**Comment by brandtbucher** [comment]
Thanks @gvanrossum, I agree on both points. Ready for another review.

**Comment by gvanrossum** [comment]
Brandt,

Your next step would be to email steering-council@python.org with a request
to review and hopefully approve this PEP.

--Guido

**Comment by gvanrossum** [comment]
> Your next step would be to email steering-council@python.org with a request
> to review and hopefully approve this PEP.

Whoops, I forgot I am already BDFL-Delegate myself. So I'll just announce that I'm accepting it.

**Pull request merged**